### ISSUE TITLE
Capture config per framework/instrumentation

### ISSUE THREAD

**Issue by pavolloffay** [hypertrace/agent-config#3]
The capture config should be configurable per framework/instrumentation but also globally (
https://github.com/hypertrace/agent-config/blob/eb054cd09abbc30e8d196618ca8526999019a5d1/config.proto#L37).

For instance I want to disable body capture for servlet but still have it enabled for okhttp client.

**Comment by jcchavezs** [comment]
While I see the consequences of the use case legit, I'd really like to run some discussion around this as I believe people is not too interested on enable/disable per library but probably per use case (e.g. client, server, remote service, etc). Any thoughts on this @davexroth @mohit-a21 ?

**Comment by pavolloffay** [comment]
The use case I had in mind was that particular body instrumentation can be problematic so it could be disabled without affecting others.

**Comment by davexroth** [comment]
Exactly, it will mostly be something used by us in the field if a particular framework causes issues in a customers application.

But as it will be more of an internal config, we may be able to have a different config block such as `message Frameworks { map of framework name to options }`
